web/gds-admin-ui/src/helpers/api/apiCore.js

```diff
@@ -1,10 +1,13 @@
 import axios from 'axios';
 import { defaultEndpointPrefix } from '../../utils';
+import jwtDecode from 'jwt-decode'
 
-// content type
 axios.defaults.headers.post['Content-Type'] = 'application/json';
 axios.defaults.baseURL = defaultEndpointPrefix();
 
+axios.defaults.withCredentials = true
```

Comment:
It might be helpful to add a comment here:

```
// withCredentials ensures that axios fetch includes `HttpOnly` cookies in the request for CSRF protectioin
```